frontend/src/components/interactive-chat-box.tsx

```diff
@@ -35,9 +35,21 @@ export function InteractiveChatBox({
     });
   };
 
+  const [inputValue, setInputValue] = React.useState(value || "");
+
+  React.useEffect(() => {
+    setInputValue(value || "");
+  }, [value]);
+
```

Comment:
```suggestion
```

The props we accept should already handling the input state (`value` instead of `inputValue` and `onChange` instead of `setInputValue`)